saleor/graphql/shop/utils.py

```diff
@@ -26,3 +28,12 @@ def get_countries_codes_list(attached_to_shipping_zones: Optional[bool] = None):
             return all_countries_codes - covered_countries_codes
 
     return all_countries_codes
+
+
+def get_track_inventory_by_default():
+    track_inventory_by_default = (
+        SiteSettings.objects.all().values("track_inventory_by_default").first()
+    )
+    if track_inventory_by_default is not None:
+        return track_inventory_by_default["track_inventory_by_default"]
+    raise NotImplementedError("Site settings not found.")
```

Comment:
Raising an error here means we need to handle it in places where this function is called, but I think we have a better option - return `None` and change usages of `get_track_inventory_by_default` to not set anything on the variant instance if `None` is returned.